client/assets/javascripts/modules_new/DetailsExpander.js

```diff
@@ -1,27 +1,32 @@
 const DetailsExpander = {
   init: function (nodes) {
-    // TODO refactor to event delegation
-    const container = nodes.querySelector('.js-details-expander')
+    const container = document.querySelectorAll('[class^="js-details-expander"],[class*=" js-details-expander"]')
```

Comment:
Yeah that was my initial approach, and this works perfectly fine on pages where there is only one instance of a conditionally revealing element.
However, on the page that was reported to have the issue, there's two elements with two separate class names (`js-details-expander-fees` and `js-details-expander-other` for example). From my understanding, you can't use wildcards or regex necessarily to capture both in this case, but from my searching the above was the only way to capture both elements.

Reply:
If the classes aren't used for anything else, I'd go through and remove the `js-details-expander-fees` and replace with `js-details-expander` in the HTML then to reduce the overall complexity.

Reply:
I had assumed that they had different class names so that once some had entered a value into the input, only the corresponding textarea would be revealed, and not both.

Although, I didn't test to confirm this.
I will have a play around to see if this works. I did add a change into the javascript which narrowed down which content to reveal based off the positioning of the elements rather than the classes. Maybe that is sufficient enough.